.github/ISSUE_TEMPLATE/issue-add-pool.yml

```diff
@@ -0,0 +1,19 @@
+name: add-pool
+description: Add your Pool to xSPO-Alliance
+title: "[add-pool]: "
+labels: [poolrequest]
+assignees:
+  - cardano-apexpool
```

Comment:
comment: needs to be a maintainer account or an account with permitions to update main branch